ldms/src/sampler/comp_id_helper.c

```diff
@@ -0,0 +1,57 @@
+#include "comp_id_helper.h"
+
+int comp_id_helper_schema_add(ldms_schema_t schema, comp_id_t cid)
+{
+	int rc;
+	if (!cid || ! schema)
+		return EINVAL;
+	if (cid->defined) {
+		rc = ldms_schema_metric_add(schema, LDMSD_COMPID, LDMS_V_U64);
```

Comment:
Unlike jobid, the component ID is not expected to change for the life of the metric set. So this should be "lmds_schema_meta_", should it not?